apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/AddressSummary.tsx

```diff
@@ -0,0 +1,120 @@
+            <div className="flex items-center  rounded-lg bg-slate-100 p-2">Optional</div>
+          )}
+        </div>
+      </div>
+      <div className="rounded-b-lg bg-white ">
+        <div className="grid h-10 grid-cols-4 items-center border-y border-slate-200 bg-slate-100 text-sm font-bold text-slate-600">
+          <div className="pl-4 md:pl-6">User</div>
+          <div className="col-span-2 pl-4 md:pl-6">Response</div>
+          <div className="px-4 md:px-6">Time</div>
+        </div>
+        {questionSummary.samples.map((response) => {
+          return (
+            <div
+              key={response.id}
+              className="grid grid-cols-4 items-center border-b border-slate-100 py-2 text-sm text-slate-800 md:text-base">
+              <div className="pl-4 md:pl-6">
+                {response.person ? (
+                  <Link
+                    className="ph-no-capture group flex items-center"
+                    href={`/environments/${environmentId}/people/${response.person.id}`}>
+                    <div className="hidden md:flex">
+                      <PersonAvatar personId={response.person.id} />
+                    </div>
+                    <p className="ph-no-capture break-all text-slate-600 group-hover:underline md:ml-2">
+                      {getPersonIdentifier(response.person)}
+                    </p>
+                  </Link>
+                ) : (
+                  <div className="group flex items-center">
+                    <div className="hidden md:flex">
+                      <PersonAvatar personId="anonymous" />
+                    </div>
+                    <p className="break-all text-slate-600 md:ml-2">Anonymous</p>
+                  </div>
+                )}
+              </div>
+              {
+                <div className="ph-no-capture col-span-2 pl-6 font-semibold">
+                  {response.value[0] && (
+                    <>
+                      {response.value[0]}
+                      <br />
+                    </>
+                  )}
+                  {response.value[1] && (
+                    <>
+                      {response.value[1]}
+                      <br />
+                    </>
+                  )}{" "}
+                  {response.value[2] && (
+                    <>
+                      {response.value[2]}
+                      <br />
+                    </>
+                  )}
+                  {response.value[3] && (
+                    <>
+                      {response.value[3]}
+                      <br />
+                    </>
+                  )}
+                  {response.value[4] && (
+                    <>
+                      {response.value[4]}
+                      <br />
+                    </>
+                  )}
+                  {response.value[5] && (
+                    <>
+                      {response.value[5]}
+                      <br />
+                    </>
```

Comment:
Can we rewrite the above potentially using a `map` method for more readability + less repeatative code?